.github/workflows/continuous-integration.yml

```diff
@@ -53,6 +53,14 @@ jobs:
             php-version: "8.0"
```

Comment:
Change existing experimental jobs instead, please. We don't need both PHP 8.0 @ dev and PHP 8.1 @ dev

Reply:
Like this?